### PR TITLE
docs: Create Code of Conduct

### LINKED ISSUE #4: Create Code of Conduct
**Is your feature request related to a problem?**

As a project within AnitaB.org Open Source Community, we need a code of conduct in the repository.

**Describe the solution you'd like**

The code of conduct should be the same as in every repo. Here is is the update one: https://github.com/anitab-org/anitab-org.github.io/blob/develop/code_of_conduct.md

**Describe alternatives you've considered**

N/A

**Additional context**

N/A

### PR DESCRIPTION
**Pull request by gaurivn**
## Description

Added a Code of Conduct file in the repository.

## Motivation and Context

Fixes #4 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by techno-disaster** [comment]
@gaurivn  i see no reason why pubspec.lock has to be updated for code of conduct, are you sure you are on the beta channel of flutter?

File: code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+## 5. Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community admins may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event).
+
+## 6. Reporting Guidelines
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community admin as soon as possible by emailing opensource@anitab.org. Please read [Reporting Guidelines](reporting_guidelines.md) for details.
```
Comment (isabelcosta) [inline code_of_conduct.md]:
```suggestion
If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community admin as soon as possible by emailing opensource@anitab.org. Please read [Reporting Guidelines](.github/reporting_guidelines.md) for details.
```

This assumes that the reporting guidelines are in the same folder. Please change this and test it on your repository, if the relative path is working

File: pubspec.lock [hunk 2]
```diff
@@ -1,13 +1,20 @@
 # Generated by pub
```
Comment (isabelcosta) [inline pubspec.lock]: Could you please remove this file from your PR changes

**Review by isabelcosta — CHANGES_REQUESTED** [review]
I left some notes, could you please address them @gaurivn . You are almost done with this PR ;)
Also, good job with having a good commit message ;) I noticed you followed the community message guidelines from the other mentorship projects 🎉

**Comment by isabelcosta** [comment]
the build showing up from the GitHub actions is very cool 👏 @Techno-Disaster

**Comment by techno-disaster** [comment]
Thanks @isabelcosta. 
@gaurivn did you copy paste the PR template manually or did it preload when you clicked new pull request?

File: code_of_conduct.md (hunk 1 above)
Comment (gaurivn) [reply code_of_conduct.md]: @Techno-Disaster , sorry, it got updated when I tried to run the app on my phone after adding code_of_conduct.md. I am changing pubspec.lock back to how it was before.

Comment (gaurivn) [reply code_of_conduct.md]: Thank you so much @isabelcosta  for your encouraging words, I will make changes soon.

Comment (techno-disaster) [reply code_of_conduct.md]: the pubspec.lock right now is what bartek was using so i might have to update it manually switching to beta channel

**Comment by gaurivn** [comment]
@Techno-Disaster , I copied the PR template manually.

**Review by techno-disaster — DISMISSED** [review]

**Review by isabelcosta — APPROVED** [review]
Great work @gaurivn 👏

**Comment by gaurivn** [comment]
Thank you so much @isabelcosta

**Review by techno-disaster — APPROVED** [review]

**Pull request merged**